Library/Homebrew/brew.sh

```diff
@@ -114,6 +114,25 @@ else
   : "${HOMEBREW_OS_VERSION:=$(uname -r)}"
   HOMEBREW_OS_USER_AGENT_VERSION="$HOMEBREW_OS_VERSION"
 
+  # Ensure the system Curl is a version that supports modern HTTPS certificates.
+  HOMEBREW_SYSTEM_CURL_VERSION="$($(type -P curl) --version 2>/dev/null | head -n1 | awk '{print $2}')"
```

Comment:
I generally prefer `command -v` over `type -P` because the former is a standard and the latter is a bashism.

Reply:
Good to know. Eventually migrating the other instances of `type -P` in the codebase might be ideal.

Reply:
@sjackman We're specifically using a lot of Bashisms to simplify the code so I think they are generally a pro rather than a con. Everywhere we `[[`: that's a Bashism.